packages/cli/src/logger/index.ts

```diff
@@ -551,9 +551,10 @@ export const logHook = (hook = '', msg = '') => {
     console.log(`${`[hook]`} ${_sanitizePaths(msg)}`);
 };
 
-export const logWarning = (msg: string | boolean | unknown) => {
+export const logWarning = (msg: string | boolean | unknown, opts?: { skipSanitizePaths?: boolean }) => {
     const ctx = getContext();
-    const msgSn = typeof msg === 'string' ? _sanitizePaths(msg) : String(msg);
+    const shouldSanitizePath = opts?.skipSanitizePaths ?? false;
+    const msgSn = typeof msg === 'string' && !shouldSanitizePath ? _sanitizePaths(msg) : String(msg);
```

Comment:
This seems to still be backwards? `if not shouldSanitizePath`, then it sanitizes paths?